test/image/mocks/gl3d_marker_symbols.json

```diff
@@ -0,0 +1,163 @@
+        "marker symbol: 'hexagram'",
+        "marker symbol: 'diamond-tall'",
+        "marker symbol: 'diamond-tall-open'",
+        "marker symbol: 'hourglass'",
+        "marker symbol: 'hourglass-open'",
+        "marker symbol: 'bowtie'",
+        "marker symbol: 'bowtie-open'",
+        "marker symbol: 'circle-cross'",
+        "marker symbol: 'circle-x'",
+        "marker symbol: 'square-cross'",
+        "marker symbol: 'square-x'",
+        "marker symbol: 'asterisk'"
+      ],
+      "marker": {
+        "color": "blue",
+        "size": 8,
+        "symbol": [
+          "circle",
+          "circle-open",
+          "square",
+          "square-open",
+          "diamond",
+          "diamond-open",
+          "cross",
+          "x",
+          "triangle-up",
+          "triangle-up-open",
+          "triangle-down",
+          "triangle-down-open",
+          "triangle-left",
+          "triangle-left-open",
+          "triangle-right",
+          "triangle-right-open",
+          "triangle-ne",
+          "triangle-ne-open",
+          "triangle-nw",
+          "triangle-nw-open",
+          "triangle-se",
+          "triangle-se-open",
+          "triangle-sw",
+          "triangle-sw-open",
+          "pentagon",
+          "pentagon-open",
+          "hexagon",
+          "hexagon-open",
+          "hexagon2",
+          "star",
+          "star-open",
+          "hexagram",
+          "diamond-tall",
+          "diamond-tall-open",
+          "hourglass",
+          "hourglass-open",
+          "bowtie",
+          "bowtie-open",
+          "circle-cross",
+          "circle-x",
+          "square-cross",
+          "square-x",
+          "asterisk"
+        ],
+        "line": {
+          "color": "orange",
+          "width": 1.5
+        }
+      }
+    }
+  ],
+  "layout": {
+    "width": 500,
+    "height": 500,
+    "scene": {
+      "camera": {
+        "eye": {
+          "x": 0.1,
+          "y": 0.1,
+          "z": 2.1
+        }
+      },
+      "xaxis": {
+        "showgrid": false,
+        "zeroline": false,
+        "showline": false,
+        "showticklabels": false,
+        "title": "",
+        "range": [0, 10]
+      },
+      "yaxis": {
+        "showgrid": false,
+        "zeroline": false,
+        "showline": false,
+        "showticklabels": false,
+        "title": "",
+        "range": [0, 6]
+      },
+      "zaxis": {
+        "showgrid": false,
+        "zeroline": false,
+        "showline": false,
+        "showticklabels": false,
+        "title": "",
+        "range": [0, 2]
+      }
+    },
+    "annotations": [
+      {
+        "x": 0.5,
+        "y": 1,
+        "xref": "paper",
+        "yref": "paper",
+        "text": "Add 100 (or '-open') to marker number (or string) to remove the fill color,<br>Add 200 (or '-dot') to marker number (or string) to add a dot in the middle,<br>Add 300 (or '-open-dot', 'dot-open') to marker number (or string) for both.",
```

Comment:
I presume `(-open)?-dot` is not supported in gl? That's fine, but we should be clear about this.